### PR TITLE
Add AssetInput component

### PR DESCRIPTION
**Pull request by trolloks**
Had to fix eslint for windows. Lint wasn't working
`"lint": "eslint \"{src,electron,e2e,}/**/*.{ts,tsx}\" --fix",`

Still busy adding coinInputAdvanced. AssetInput is dependent on it.

### CODE REVIEW

**Comment by nomadnic** [comment]
Closes #91

File: src/components/uielements/assets/assetInput/assetInput.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+import AssetInput from './assetInput'
+
+storiesOf('Components/Assets/AssetInput', module).add('default', () => {
+  return (
+    <div style={{ padding: '20px' }}>
+      <AssetInput
+        title="swap amount"
+        status="slip 2%"
+        amount={tokenAmount(12345).amount()}
```
Comment (veado) [inline src/components/uielements/assets/assetInput/assetInput.stories.tsx:14]: [hint] You can use `bn` from `@thorchain/asgardex-util` to create a `BigNumber` more easily: 
```
bn(12345)
```

File: src/components/uielements/assets/assetInput/assetInput.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+
+import { storiesOf } from '@storybook/react'
+import { tokenAmount } from '@thorchain/asgardex-token'
```
Comment (veado) [inline src/components/uielements/assets/assetInput/assetInput.stories.tsx:4]: `@thorchain/asgardex-token` has been deprecated. Use `@thorchain/asgardex-util`, which includes all of `asgardex-token`, but renamed its function from `tokenXXX` to `assetXXX`

File: src/components/uielements/assets/assetInput/assetInput.tsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+  const onChangeHandler = useCallback(
+    (value: BigNumber) => {
+      onChange(value)
+    },
+    [onChange]
+  )
+
+  const handleClickWrapper = useCallback(() => {
+    inputRef.current.firstChild.focus()
```
Comment (veado) [inline src/components/uielements/assets/assetInput/assetInput.tsx:33]: `current` + `firstChild` might be `undefined`.
```
inputRef.current?.firstChild?.focus()
```

**Pull request merged**